app/javascript/components/patient/Patient.js

```diff
@@ -727,6 +744,7 @@ Patient.propTypes = {
   can_modify_subject_status: PropTypes.bool,
   authenticity_token: PropTypes.string,
   workflow: PropTypes.string,
+  priority: PropTypes.number,
```

Comment:
I think calling this `headingLevel` would make it clearer what it is

Reply:
fixed.